packages/gestalt/src/AnimationController.js

```diff
@@ -150,12 +158,14 @@ function AnimationController({
     }
   }, [animationState, onDismissEnd]);
 
+  const contextValue = useMemo(() => ({ animationState, setAnimationState }), [animationState]);
```

Comment:
Basically the same as the rule about unstable functions — this makes sure the context value is stable